polkadot/primitives/src/lib.rs

```diff
@@ -19,8 +19,8 @@
 #![warn(missing_docs)]
 #![cfg_attr(not(feature = "std"), no_std)]
 
-// `v6` is currently the latest stable version of the runtime API.
-pub mod v7;
+// `v11` is currently the latest stable version of the runtime API.
```

Comment:
Not introduced here, but not sure why we need to mentione runtime API version here.